server/main.go

```diff
@@ -71,6 +72,29 @@ func addPlayer(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, &gameState)
 }
 
+type addPromptsRequest struct {
+	PlayerName string `json:"playerName"`
+	GroupName  string `json:"groupName"`
+	Noun       string `json:"noun"`
+	Adjective1 string `json:"adjective1"`
```

Comment:
Nit: We could just have an array of adjectives vs `adjective1`and `adjective2`

Reply:
Sounds good.